components/rich_text_editor/extensions/emoji/EmojiComponent.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <node-view-wrapper class="d-d-inline-block">
+    <dt-emoji
+      size="200"
+      :code="node.attrs.code"
+    />
+  </node-view-wrapper>
```

Comment:
Hmm so when the screenreader reads this emoji in the input it doesn't read it's proper description ex "grinning face with smiling eyes" instead it just says "object replacement character". The description is correctly set on the emoji but it's also wrapped with `data-node-view-wrapper` perhaps this is what's preventing it from being read?

Reply:
that node viewer wrapper is needed for custom node viewer. let me check docs on how to go about this. thanks!

Reply:
This issue is too complex to fix atm. We can log and try to fix later. Even slacks message input does not do this correctly.